tests/storage/hotplug.go

```diff
@@ -318,7 +318,7 @@ var _ = SIGDescribe("Hotplug", func() {
 
 	verifyCreateData := func(vmi *kubevirtv1.VirtualMachineInstance, device string) {
 		batch := []expect.Batcher{
-			&expect.BSnd{S: fmt.Sprintf("sudo mkfs.ext4 %s\n", device)},
+			&expect.BSnd{S: fmt.Sprintf("sudo mkfs.ext4 -F %s\n", device)},
```

Comment:
This is a bad thing to do, -F forces a file system if one already exists, this usually means we are re-using a disk and its a bug.

Reply:
Right, I will change that. Now its just a temporary fix for the new test with migration. The test starts a VM, attaches a volume and then does several migrations. Each time a migration is completed it checks whether the volume is still usable. I will refactor it to create the filesystem only once.